doc/tae.rst

```diff
@@ -0,0 +1,50 @@
+.. _branin: quickstart.html#branin
```

Comment:
I think it would be great if there were some links to the introductory examples on how to actually use the target algorithm evaluators.